docs/descriptors/xml_xmllint.md

```diff
@@ -25,7 +25,7 @@ To apply file formatting you must set `XML_XMLLINT_CLI_LINT_MODE: file` and `XML
 | Variable                                | Description                                                                                                                                                                                                         | Default value      |
 |-----------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------|
 | XML_XMLLINT_AUTOFORMAT                  | If set to `true`, it will reformat and reindent the output                                                                                                                                                          | `false`            |
-| XML_XMLLINT_INDENT                      | The number of indentation spaces when `XML_XMLLINT_AUTOFORMAT` is `true`                                                                                                                                            | `  `               |
+| XML_XMLLINT_INDENT                      | The number of indentation spaces when `XML_XMLLINT_AUTOFORMAT` is `true`                                                                                                                                            | ``                 |
```

Comment:
This auto formatting seems problematic, but is not present after a force-push, maybe merged by another PR as is. It does not mean the same thing.

Reply:
Is there other places where this happens?

Reply:
It's the build doc & autofix that fight each other, happens for months , maybe years, it's not nice but not a big deal ^^

What is more wrong is that I forced the merge of a tiny PR and I didn't think that latest version of checkov would be broken :/

Reply:
Posted a bug... :/ 

https://github.com/bridgecrewio/checkov/issues/5841